findme-users/src/FollowerRepository.py

```diff
@@ -0,0 +1,162 @@
+from datetime import datetime
+
+import boto3
+from aws_lambda_powertools.event_handler.exceptions import (
+    BadRequestError,
+)
+from boto3.dynamodb.conditions import Attr
+from botocore.exceptions import ClientError
+from pydantic import ValidationError
+
+
+class FollowerRepository:
+    def __init__(self):
+        self.dynamodb = boto3.resource('dynamodb', region_name='eu-central-2')
+        self.table = self.dynamodb.Table('FollowerTable')
+
+    def create_follow_request(self, follow_request):
+        try:
+            # Check if follow request already exists
+            existing_request = self.table.scan(
+                FilterExpression=Attr('partition_key').eq(
+                    f"REQUEST#{follow_request.requester_id}#{follow_request.requestee_id}") & Attr('sort_key').eq(
```

Comment:
Use & Key('sort_key') -> The sortkey is not only a simple attribute

Reply:
Same for partition_key

Reply:
Resolving this conversation since it was fixed by @chrsBa in 6c148977687263526bfb0dfe61d2d29560bf756d